### ISSUE TITLE
notify on last update failure

### ISSUE THREAD

**Issue by huglester** [martincyprus/wtc-miner-monitor#4]
Hello,
so if we got 15 peers (ie. rig hungs) and not updateing stats, we can see this:
![image](https://user-images.githubusercontent.com/150731/41450060-ddb3342c-706d-11e8-806e-a4d510b6ac26.png)

so peers are ok, but miners is not mining, and we do not get telegram notifications

**Comment by martincyprus** [comment]
well that looks like its been down for 5 minutes, then it should send you a telegram message, assuming telegram is configured correctly..
you can test the telegram settings by starting server with:

server.exe test

that will test your telegram configs..

also I think something is wrong, you are pointing to the wrong 20021 file, 10082461 hashes/s is very unlikely to be on all your miners.. so you are for sure pointing to the wrong 200021 file..

I would advise in your startup scripts to go into the folder (the gpumining_v0.2/) before starting the ming_runXX.exe

that would make sure the 20021 file is written in there and unique per GPU..
contact me on slack or telegram.. for more fast response..

**Comment by huglester** [comment]
Yes hashes should not be same. I also wondered about this. I receive telegram notifications about 0 peers. But no notifications when last update was like 1hr ago. 

I will test and will contact you

**Comment by martincyprus** [comment]
soo the going down notifications, there is one or two notifications that happen around the 5-8 minute mark of being down.. then there is a another notification around the 56-59 minute mark of being down.. but that is it..  it was considered a bit too spammy to send all the time.. for the 0 peer though its very spammy.. its using a different path to determine when to send.. its not ideal..

**Comment by huglester** [comment]
Then its strange. 
- i get notifications for same nodes every minute when there are 0 peers
- I do not get notification when nodes are not updating (even hour or more)

**Comment by huglester** [comment]
Hello,
so I have figured out. My hashes are changing now.

But I get notification only when peers are 0. and they are going every minute.
If, for example, blocks, or last update are long ago (http://pic.webas.lt/webas.lt-e2a5eb92db.png) - I do not get this notifications.
